### ISSUE TITLE
Events Page Edits

### ISSUE THREAD

**Issue by katalinadeaven** [engagingnewsproject/enp-platform#118]
- [ ] Change "previous events" page so the most recent events show up at the top: https://mediaengagement.org/events/list/?tribe_paged=1&tribe_event_display=past&tribe-bar-date=2021-01-14

**Comment by leoncai1999** [comment]
I have currently opened a PR for this task here: https://github.com/engagingnewsproject/enp-platform/pull/123

**Comment by luukee** [comment]
@leoncai1999 Hey Leon, just pushed the events changes to the [staging site](https://cmestaging.wpengine.com/events/list/?tribe_paged=1&tribe_event_display=past&tribe-bar-date=2021-01-29). Wouldn't we want the most recent events at the top of the page? Or are there still changes that need to happen on WP Dashboard?

**Comment by leoncai1999** [comment]
A follow-up PR has been opened to resolve this issue: https://github.com/engagingnewsproject/enp-platform/pull/124

**Comment by leoncai1999** [comment]
I have now created a PR that reorders all past events (both when we're viewing a subset of past events and when we're viewing all past events on one page. #129 ).  That PR has been merged in!

We would have liked to have a button on the top search/filter bar that would take us to the view of all past events. There is a way to alter the elements on that bar, however, that's an additional feature that I believe we have to pay for. If we'd like to find another way to go about doing this, I'm happy to look into it in the future!
